localization/autoware_pose_covariance_modifier/README.md

```diff
@@ -0,0 +1,232 @@
+| `output/gnss_position_stddev`       | `std_msgs::msg::Float64`                        | Output pose gnss average standard deviation in position xy. It is published only when the enable_debug_topics is true. |
+
+### Parameters
+
+The parameters are set
+in [config/autoware_pose_covariance_modifier.param.yaml](config/autoware_pose_covariance_modifier.param.yaml) .
+
+#### Standard deviation thresholds
+
+{{ json_to_markdown("
+localization/autoware_pose_covariance_modifier_node/schema/sub/stddev_thresholds.sub_schema.json") }}
+
+#### Validation
+
+{{ json_to_markdown("localization/autoware_pose_covariance_modifier_node/schema/sub/validation.sub_schema.json") }}
+
+#### Debug
+
+{{ json_to_markdown("localization/autoware_pose_covariance_modifier_node/schema/sub/debug.sub_schema.json") }}
+
+## FAQ
+
+### How are varying frequency rates handled?
+
+The GNSS and NDT pose topics may have different frequencies.
+The GNSS pose topic may have a higher frequency than the NDT.
+
+Let's assume that the inputs have the following frequencies:
+
+| Source | Frequency |
+| ------ | --------- |
+| GNSS   | 200 Hz    |
+| NDT    | 10 Hz     |
+
+This package publishes the output poses as they come in, depending on the mode.
+
+End result:
+
+| Mode       | Output Freq |
+| ---------- | ----------- |
+| GNSS Only  | 200 Hz      |
+| GNSS + NDT | 210 Hz      |
+| NDT Only   | 10 Hz       |
+
+### How and when are the NDT covariance values overwritten?
+
+| Mode       | Outputs, Covariance                         |
+| ---------- | ------------------------------------------- |
+| GNSS Only  | GNSS, Unmodified                            |
+| GNSS + NDT | **GNSS:** Unmodified, **NDT:** Interpolated |
+| NDT Only   | NDT, Unmodified                             |
+
+NDT covariance values overwritten only for the `GNSS + NDT` mode.
+
+This enables a smooth transition between `GNSS Only` and `NDT Only` modes.
+
+In this mode, both NDT and GNSS poses are published from this node.
+
+#### Assumptions
+
+- Incoming NDT poses have a fixed covariance value.
+- Covariance provided by GNSS is reliable.
+
+#### NDT covariance calculation
+
+##### Goal
+
+- As the `gnss_std_dev` increases within its bounds, `ndt_std_dev` should proportionally decrease within its own bounds.
+
+To achieve this, we first linearly interpolate:
+
+- Base value: `gnss_std_dev`
+- Base range: [`threshold_gnss_stddev_xy_bound_lower`, `threshold_gnss_stddev_xy_bound_upper`]
+- Target range: [`ndt_std_dev_bound_lower`, `ndt_std_dev_bound_upper`]
+- Target value: `ndt_std_dev_target`
+
+- Final value = `ndt_std_dev_bound_lower` + `ndt_std_dev_bound_upper` - `ndt_std_dev_target` (to get the inverse)
+
+#### Illustration
+
+<img width="300" src="media/range_lerp.svg" alt="range to range lerp animation">
```

Comment:
I think the subsection headings "Goal" and "Illustration" are redundant so removing them will make it clear to read. (The content itself is need to be here.)